pkg/cluster/check/default.go

```diff
@@ -41,7 +41,7 @@ func DefaultClusterChecks() []ClusterCheck {
 		func(cluster ClusterInfo) conditions.Condition {
 			return conditions.PollingCondition("apid to be ready", func(ctx context.Context) error {
 				return ApidReadyAssertion(ctx, cluster)
-			}, 2*time.Minute, 5*time.Second)
+			}, 5*time.Minute, 5*time.Second)
```

Comment:
this is much needed fix!

tl;dr is that there's no wait for `bootkube` anymore, so we need to wait more for worker node kubelets to join and get certs